target/linux/bcm27xx/image/config.txt

```diff
@@ -12,3 +12,8 @@ include distroconfig.txt
 
 [all]
 # Place your custom settings here.
+# usb_max_current_enable=1
+# Enable the PCIe external connector
+# dtparam=pciex1
+# Force Gen 3.0 speeds
+# dtparam=pciex1_gen=3
```

Comment:
These custom settings are meant for the users, so you shouldn't change this file.
Also, PCIe 3.0 speeds aren't certified and may cause unstability: https://www.raspberrypi.com/documentation/computers/raspberry-pi-5.html#pcie-gen-3-0

> WARNING
> The Raspberry Pi 5 is not certified for Gen 3.0 speeds, and connections to PCIe devices at these speeds may be unstable.

Reply:
Ok dropped the custom setting